snuba/request/schema.py

```diff
@@ -109,7 +109,7 @@ def validate(
         self, value: MutableMapping[str, Any], dataset: Dataset, referrer: str
     ) -> Request:
         try:
-            value = validate_jsonschema(value, self.__composite_schema)
+            value = validate_jsonschema(value, self.__composite_schema)  # type: ignore
```

Comment:
reason ?

Reply:
Actually I see `validate_jsonschema` is untyped. Please remove the ignore. When we will add type hints there it will get fixed, otherwise, when we will type hint that method we will not see that this is ignored and we will forget about it.

Reply:
I type hinted the method. Best of both worlds.